updater/aws_test.go

```diff
@@ -90,7 +90,8 @@ func TestSendCommandSuccess(t *testing.T) {
 	commandID, err := u.sendCommand(instances, "test-doc")
 	require.NoError(t, err)
 	assert.EqualValues(t, "command-id", commandID)
-	assert.Equal(t, instances, waitInstanceIDs)
+	assert.Contains(t, waitInstanceIDs, "inst-id-2")
+	assert.Contains(t, waitInstanceIDs, "inst-id-1")
```

Comment:
https://pkg.go.dev/github.com/stretchr/testify/assert#ElementsMatch
Same suggestion can be applied to similar assertions in the file .
```suggestion
	assert.ElementsMatch(t, instances, waitInstanceIDs)
```

Reply:
Oh nice, I somehow didn't come across/notice `ElementsMatch` when looking through the different assertions for something better than `Contains`. Thanks for pointing that out!